fedora/fas.py

```diff
@@ -100,19 +100,7 @@ async def group_info(self, evt: MessageEvent, groupname: str) -> None:
             f"**Chat:** {chat_channels}{NL}"
         )
 
-    @command.new(help="Return brief information about a Fedora user.", aliases=ALIASES["hello"])
-    @command.argument("username", pass_raw=True, required=False)
-    async def hello(self, evt: MessageEvent, username: str) -> None:
-        """
-        Returns a short line of information about the user. If no username is provided, defaults to
-        the sender of the message.
-
-        #### Arguments ####
-
-        * `username`: A Fedora Accounts username or a Matrix User ID
-           (e.g. @username:fedora.im)
-
-        """
+    async def _user_hello(self, evt: MessageEvent, username: str) -> None:
```

Comment:
If `username` can be `None`, let's state it in the type hints: `username: str | None`